packages/home-service/src/resolver.ts

```diff
@@ -1,31 +1,44 @@
 import { Home } from './schema';
-import { pubsub } from './helpers';
-
-const Home_UPDATE = 'Home_UPDATE';
 
 export const HomeResolver = {
   Query: {
     // queries
```

Comment:
Organize the comments well always put comment above the proposed update

Reply:
Noted and changed